src/Traits/JsonSerializer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Softonic\GraphQL\Traits;
+
+trait JsonSerializer
+{
+    public function jsonSerialize(): array
+    {
+        $item = [];
```

Comment:
Should the default value be `null` or `[]`? Why?

Reply:
I prefer [], but it could be null as well. There's no reason to choose one or the other.